app/ui/tooltip.d.ts

```diff
@@ -0,0 +1,2 @@
+declare function _exports(tooltipText: string, action?: Function, opts: any): any;
```

Comment:
`app/ui` shouldn't exist. Move type definition to proper location.

Reply:
done